tests/backup/inventory_test.js

```diff
@@ -264,3 +265,36 @@ Scenario(
     I.see(backupName, backupInventoryPage.elements.fullBackUpName);
   },
 );
+
+Scenario(
+  'PMM-T1163 Verify that Backup time format is identical for whole feature @backup @alyona-p',
+  async ({
+    I, backupInventoryPage, backupAPI, scheduledAPI, scheduledPage,
+  }) => {
+    // Every 2 mins schedule
+    const schedule = {
+      service_id: serviceId,
+      location_id: locationId,
+      cron_expression: '*/2 * * * *',
+      name: 'PMM-T1163 schedule',
+      mode: scheduledAPI.backupModes.snapshot,
+      description: '',
+      retry_interval: '30s',
+      retries: 0,
+      enabled: true,
+      retention: 1,
+    };
+    const scheduleId = await scheduledAPI.createScheduledBackup(schedule);
+
+    await backupAPI.waitForBackupFinish(null, schedule.name, 240);
+
+    const backupDate = moment().format('YYYY-MM-DDHH:mm:00');
```

Comment:
this way the test may be unstable. The timestamp may vary depending on a time

Reply:
added new method